src/styles.ts

```diff
@@ -57,29 +99,45 @@ export class StyleCompiler extends EventEmitter {
       }));
   }
 
-  private postprocess(processed: any) {
+  private postprocess(output: string, processed: any) {
     return new Promise((resolve, reject) => {
-      postcss(this.config.postcss.map(([pluginName, pluginConfig]) =>
-        require(pluginName)(pluginConfig)))
-        .process(processed.css, {map: {inline: false}, to: this.config.output})
+      postcss((this.config.postcss || []).map(([pluginName, pluginConfig]) => {
+        const plugin = require(pluginName);
+        if (pluginConfig) {
+          return plugin(pluginConfig);
+        }
+        return plugin;
+      }))
+        .process(processed.css, {
+          map: this.config.sourceMaps && {inline: false},
```

Comment:
It is larger ... external source maps can also be used in production builds, if you like.